openedx_tagging/core/tagging/import_export/api.py

```diff
@@ -0,0 +1,34 @@
+from io import BytesIO
+from typing import List
+
+from ..models import Taxonomy
+from .parsers import get_parser, ParserFormat
+from .exceptions import ImportError
+from .dsl import TagImportDSL
+
+# TODO This function must be a celery task
```

Comment:
It's ok to have the logic in the python API, and just wrap the call to this method in a celery task.